packages/firebase-auth/vitest.config.ts

```diff
@@ -20,20 +21,19 @@ const firebasePlugin = {
   },
 } satisfies Plugin
 
-export default defineWorkersProject({
+export default defineProject({
   test: {
     globals: true,
     include: ['src/**/*.test.ts'],
-    poolOptions: {
-      workers: {
-        miniflare: {
-          compatibilityDate: '2025-03-10',
-          compatibilityFlags: ['nodejs_compat'],
-          kvNamespaces: ['PUBLIC_JWK_CACHE_KV'],
-        },
-      },
-    },
   },
-
-  plugins: [firebasePlugin],
+  plugins: [
+    cloudflareTest({
+      miniflare: {
+        compatibilityDate: '2025-03-10',
+        compatibilityFlags: ['nodejs_compat'],
+        kvNamespaces: ['PUBLIC_JWK_CACHE_KV'],
+      },
+    }),
```

Comment:
The plugin accepts the same options as the old `poolOptions.workers` configuration